Sprint-2/extend/format-time.js

```diff
@@ -16,9 +22,31 @@ console.assert(
   `current output: ${currentOutput}, target output: ${targetOutput}`
 );
 
-const currentOutput2 = formatAs12HourClock("23:00");
-const targetOutput2 = "11:00 pm";
+const currentOutput2 = formatAs12HourClock("22:00");
+const targetOutput2 = "10:00 pm";
 console.assert(
   currentOutput2 === targetOutput2,
   `current output: ${currentOutput2}, target output: ${targetOutput2}`
 );
+
+const currentOutput3 = formatAs12HourClock("00:00");
+const targetOutput3 = "12:00 am";
+console.assert(
+  currentOutput3 === targetOutput3,
+  `current output: ${currentOutput3}, target output: ${targetOutput3}`
+);
+
+
+const currentOutput4 = formatAs12HourClock("14:00");
+const targetOutput4 = "02:00 pm";
+console.assert(
+  currentOutput4 === targetOutput4,
+  `current output: ${currentOutput4}, target output: ${targetOutput4}`
+);
+
```

Comment:
Could also test borderline cases like "12:00", "23:59".

Reply:
I've added assertions to test edge cases : "12:00" and "23:59".